consumer.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import os\n",
+    "import sys\n",
+    "import boto3\n",
+    "import matplotlib.pyplot as plt\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "parent_dir = os.path.abspath(os.path.join(os.getcwd(), os.pardir))\n",
+    "sys.path.append(parent_dir)\n",
+    "\n",
+    "load_dotenv()\n",
+    "\n",
+    "from utils.helper_methods import (\n",
+    "    redshift_read_table,\n",
+    "    read_s3_data,\n",
+    "    get_redshift_endpoint,\n",
+    ")\n",
+    "\n",
+    "redshift_master_user = os.environ.get(\"REDSHIFT_MASTER_USER\")\n",
+    "redshift_master_password = os.environ.get(\"REDSHIFT_MASTER_PW\")\n",
+    "redshift_db_name = os.environ.get(\"REDSHIFT_DB_NAME\")\n",
+    "redshift_cluster_name = os.environ.get(\"REDSHIFT_CLUSTER_NAME\")\n",
+    "redshift_table_name = os.environ.get(\"REDSHIFT_TABLE_NAME\")\n",
+    "\n",
+    "bucket_name = os.environ.get(\"S3_BUCKET_NAME\")\n",
+    "\n",
+    "cluster_address, cluster_port = get_redshift_endpoint(redshift_cluster_name)\n",
+    "\n",
+    "connection_string = f\"dbname={redshift_db_name} user={redshift_master_user} password={redshift_master_password} host={cluster_address} port={cluster_port}\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Verify raw intermediate data in S3 bucket\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_client = boto3.client(\"s3\")\n",
+    "bucket_data = read_s3_data(\n",
+    "    s3_client,\n",
+    "    \"firehose-raw-data\",\n",
+    ")\n",
+    "\n",
+    "print(len(bucket_data))\n",
+    "bucket_data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bucket_data.values()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to and read from Redshift Database\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# table_name = \"STL_LOAD_ERRORS\"\n",
```

Comment:
leftover comment?

Reply:
removed in: https://github.com/localstack-samples/sample-cdk-kinesis-firehose-redshift/pull/2/commits/1fee2753c9179974c4d98f8b0ff70ace8ba8110e